src/main/java/Storage.java

```diff
@@ -0,0 +1,78 @@
+import java.io.*;
```

Comment:
Perhaps listing out all the imports might be better according to the coding standards?